rtc_types/src/byte_formats/rkyv_format.rs

```diff
@@ -0,0 +1,105 @@
+//! [`rkyv`] byte format helpers.
+
+use core::mem::size_of;
+
+use rkyv::{
+    archived_root,
+    ser::{
+        serializers::{BufferSerializer, BufferSerializerError},
+        Serializer,
+    },
+    Aligned, Archive, Deserialize, Infallible, Serialize, Unreachable,
+};
```

Comment:
This is a minor code-aesthetic thing, so feel free to disregard.

For imports I prefer having a flatter structure. For example:
```
use rkyv::{
    archived_root, Aligned, Archive, Deserialize, Infallible, Serialize, Unreachable,
};
use rkyv::ser::Serializer;
use rkyv::ser::serializers::{BufferSerializer, BufferSerializerError},
```
This is not important, so feel free to mark this as resolved. Just wanted to mention it.

Reply:
I like that style too!

I was using the nested format only because the rkyv examples tend to, and because IntelliJ Rust has an action for automatically nesting them like that.

Unfortunately, the automatic formatting only seems to support fully nested (or top-level nested) and fully flat… not module-level nested. I'm not particularly keen to do manual formatting.

However, I looked into it, and `rustfmt` actually has an option to format imports like this, as well as doing logical grouping and such! I made a PR for that (#95), but I'll hold off on committing the project-wide changes until the current batch of PRs are in.

Reply:
Awesome thanks!